src/transformers/modeling_tf_electra.py

```diff
@@ -592,6 +607,197 @@ def call(
         return output  # (masked_lm_loss), prediction_scores, (hidden_states), (attentions)
 
 
+@add_start_docstrings(
+    """ELECTRA Model with a sequence classification/regression head on top (a linear layer on top of
+    the pooled output) e.g. for GLUE tasks. """,
+    ELECTRA_START_DOCSTRING,
+)
+class TFElectraForSequenceClassification(TFElectraPreTrainedModel, TFSequenceClassificationLoss):
+    def __init__(self, config, *inputs, **kwargs):
+        super().__init__(config, *inputs, **kwargs)
+        self.num_labels = config.num_labels
+        config.summary_proj_to_labels = True
```

Comment:
This should be removed if `"summary_proj_to_labels": true` is added to the config file.